src/sentry/notifications/notifications/digest.py

```diff
@@ -80,7 +81,8 @@ def get_title_link(self, recipient: Team | User) -> str | None:
     def build_attachment_title(self, recipient: Team | User) -> str:
         return ""
 
-    def get_reference(self) -> Any:
+    @property
+    def reference(self) -> Model | None:
```

Comment:
@mgaeta Similar here. But maybe Python isn't smart enough to know that the Project model inherits from the base model.